config/crd/bases/datasciencepipelinesapplications.opendatahub.io_datasciencepipelinesapplications.yaml

```
@@ -748,6 +748,12 @@ spec:
                         type: object
                     type: object
                 type: object
+              podToPodTLS:
+                default: true
+                description: PodToPodTLS Set to "true" or "false" tls communication
+                  between DSPA components. Default to enable Inter-Pod tls by setting
+                  to "true". Only supported in DSP V2 on OpenShift.
```

sorry, I missed this typo before

```suggestion
                description: PodToPodTLS Set to "true" or "false" to enable or disable TLS communication
                  between DSPA components (pods). Defaults to "true" to enable TLS between all pods. Only supported in DSP V2 on OpenShift.
```

---

fixed